runtime-infra/pn-ipc.yaml

```diff
@@ -283,6 +357,81 @@ Resources:
               - !GetAtt "SafeStorageToDeliveryPushQueue.Outputs.QueueARN"
               - !GetAtt "SafeStorageToDowntimeLogsQueue.Outputs.QueueARN"
 
+  # Log all events
+  CoreEventBusEventsLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: /aws/events/core-event-bus-input-events
+      RetentionInDays: 14
+
+  CoreEventBusMatchedEventsLogGroup:
```

Comment:
Come si fa a capire quale regola ha fatto match con l'evento ?

Reply:
@marco-vit-pagopa ho creato un input transformer; purtroppo con questo approccio si devono mappare uno ad uno i dettagli dell'evento che si vuole loggare in quanto il target cloudwatch si aspetta una stringa nel campo `message`. Avendo l'ID dell'evento, comunque, è possibile risalire al payload completo dal log group che traccia tutti gli eventi del core event bus tramite questa ricerca cloudwatch `{ $.id = "4270db72-d6b8-20c0-1679-f749e31ab36e" }`  (sostituendo opportunamente l'ID).